.tekton/pull-request.yaml

```diff
@@ -117,11 +117,14 @@ spec:
               imagePullPolicy: Always
               args: [
                 "--ginkgo.label-filter=build-templates-e2e",
-                "--ginkgo.progress",
                 "--ginkgo.v",
                 "--ginkgo.no-color"
               ]
               env:
+              - name: APP_SUFFIX
+                value: "{{ pull_request_number }}"
```

Comment:
By default, when the test fails, we are keeping application and component CRs in the test namespace (in case the test passes, all related resources - application CR, component CR, pipelineruns - are deleted).

This env var tells e2e test to create an application CR with a name that has a specific suffix (PR number), instead of randomly generating it
In case such application CR exists (from previous failed PR test), it will first delete the CR and recreates it.

This way we can avoid polluting the test namespace with resources generated by e2e test (because once the PR test passes, all related resources will be deleted from the namespace)